ff-netF48-server-sdk.csproj

```diff
@@ -33,12 +33,26 @@
         <None Remove="tests\**;examples\**" />
     </ItemGroup>
 
+    <!--
+        When updating a package to the latest version, order is important when linking the dependency library to a specific target .NET version and newer versions should be referenced first.
+
+        See "System.IdentityModel.Tokens.Jwt".
+    -->
+
     <ItemGroup>
+        <!-- Unless proven otherwise, 4.0.0 catch all will not work, please add a specific reference for your FTFM package, preferably targeting latest. -->
+        <PackageReference Include="Disruptor" Version="6.0.0" Condition="'$(TargetFramework)' == 'net8.0'" />
         <PackageReference Include="Disruptor" Version="4.0.0" />
+
         <PackageReference Include="murmurhash" Version="1.0.3" />
         <PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
         <PackageReference Include="System.ComponentModel.Annotations" Version="5.0.0" />
-        <PackageReference Include="System.IdentityModel.Tokens.Jwt" Version="6.34.0" />
+
+        <!-- Unless proven otherwise, 6.36.0 catch all will not work, please add a specific reference for your FTFM package, preferably targeting latest. -->
```

Comment:
Someone adding `net10.0` won't see this comment until it's too late, unless you put a `Condition = x or y or z for all bute net7 and net8` on the catchall...

Reply:
Updated the conditions on the catch all libraries.